src/main/java/org/testcontainers/containers/TarantoolImageParams.java

```diff
@@ -38,6 +39,17 @@ public TarantoolImageParams(String tag, File dockerfile, Map<String, String> bui
         this.buildArgs = buildArgs;
     }
 
+    /**
+     * Custom constructor for tarantool image parameters
+     *
+     * @param tag          docker image tag
+     * @param dockerfile   dockerfile for building custom tarantool image
+     * @param defaultImage default image name for building docker image
+     */
+    public TarantoolImageParams(String tag, File dockerfile, String defaultImage) {
```

Comment:
Так кажется понятнее

```suggestion
    public TarantoolImageParams(String tag, File dockerfile, String baseImageName) {
```